website/docs/docs/core/connect-data-platform/spark-setup.md

```diff
@@ -41,7 +41,7 @@ See [Databricks setup](#databricks-setup) for the Databricks version of this pag
 
 pip is the easiest way to install the adapter:
 
-<code>pip install {frontMatter.meta.pypi_package}</code>
+<code>python -m pip install {frontMatter.meta.pypi_package}</code>
 
 <p>Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> and any other dependencies.</p>
```

Comment:
There's a lot of duplication here across adapters. Would this be a good candidate for a snippet?

If so, seems like we'd want to tackle that within a separate issue.

Reply:
I'll take it up in an issue and review. Thanks!

Reply:
Looks like this was already taken care of!